numeric.rb

```diff
@@ -323,44 +309,37 @@ def magnitude
     Primitive.cexpr! 'rb_float_abs(self)'
   end
 
-  #
   # call-seq:
-  #    -float  ->  float
+  #   -float -> float
   #
-  # Returns +float+, negated.
+  # Returns +self+, negated.
   #
   def -@
     Primitive.attr! :leaf
     Primitive.cexpr! 'rb_float_uminus(self)'
   end
 
-  #
   #  call-seq:
-  #     float.zero?  ->  true or false
-  #
-  #  Returns +true+ if +float+ is 0.0.
+  #    zero? -> true or false
   #
+  #  Returns +true+ if +self+ is 0.0, +false+ otherwise.
   def zero?
     Primitive.attr! :leaf
     Primitive.cexpr! 'RBOOL(FLOAT_ZERO_P(self))'
   end
 
-  #
   #  call-seq:
-  #     float.positive?  ->  true or false
-  #
-  #  Returns +true+ if +float+ is greater than 0.
+  #    positive? -> true or false
   #
+  #  Returns +true+ if +self+ is greater than 0, +false+ otherwise.
   def positive?
     Primitive.attr! :leaf
     Primitive.cexpr! 'RBOOL(RFLOAT_VALUE(self) > 0.0)'
   end
 
-  #
   #  call-seq:
-  #     float.negative?  ->  true or false
+  #    negative? -> true or false
   #
-  #  Returns +true+ if +float+ is less than 0.
```

Comment:
I think you deleted the old documentation without adding a new one.

Reply:
Indeed.  Fixed.